File: ProgramasJava/src/br/com/evertonsantos/secao07/Exercicio04.java
package br.com.evertonsantos.secao07;

import java.util.Scanner;

public class Exercicio04 {

	public static void main(String[] args) {
		int num, maior = -999, menor = 999, soma = 0;
		float media;
		Scanner teclado = new Scanner(System.in);
		
		for (int i = 0; i < 10; i++) {
			System.out.println("Informe um número inteiro positivo: ");
			num = teclado.nextInt();
			
			if(num > 0) {
			
				if (num > maior) {
				maior = num;
				}			
				if (num < menor) {
				menor = num;
				}
				soma = soma + num;
		
			}else {
				i--;
			}			
			
	}
	media = soma / 10;
	System.out.println("O maior número é: " +maior);
	System.out.println("O menor número é: " +menor);
	System.out.println("A média dos números é: " +media);
	teclado.close();	
		
	
	}
}	

File: ProgramasPython/secao07/exercicio04.py
maior = -999
menor = 999
soma = 0

for n in range(1,11):
    num = int(input("Informe um número: "))
    
    while num < 0:
        num = int(input("Informe um número: "))
    
    if num > maior:
        maior = num
    if num < menor:
        menor = num
    soma = soma + num
    
media = soma / 10

print("O maior número é: {0}".format(maior))
print("O menor número é: {0}".format(menor))
print("A média dos números é: {0}".format(media))